internal/appsec/appsec.go

```diff
@@ -11,9 +11,10 @@ import (
 
 	"github.com/DataDog/appsec-internal-go/limiter"
 	appsecLog "github.com/DataDog/appsec-internal-go/log"
+	waf "github.com/DataDog/go-libddwaf/v2"
+
 	"github.com/DataDog/dd-trace-go/v2/internal/appsec/dyngo"
 	"github.com/DataDog/dd-trace-go/v2/internal/log"
-	waf "github.com/DataDog/go-libddwaf/v2"
 )
```

Comment:
This change (and the one in `internal/appsec/config.go`) look like we might be using disjoint import formatter settings... I'd suggest we stick to `goimports` or something?

Reply:
I defer to whatever the CI says. Agree, though, I'd be happy to use `goimports`. Let's talk about it in the guild.